guides/common/modules/proc_rex-pull-based-performance-impact-considerations.adoc

```diff
@@ -0,0 +1,32 @@
+[id="REX_pull_based_performance_impact_considerations_{context}"]
+= REX pull-based transport performance impact considerations
+
+When {ProjectServer} is configured to use REX pull-based transport for remote execution jobs using "Script" provider, {SmartProxyServer} sends a notification about new job to the client machines via MQTT.
+This notification does not include actual workload (actual script) the client is supposed to execute.
```

Comment:
Is this content partially identical to guides/common/modules/proc_decreasing-performance-impact-of-the-pull-based-rex-transport.adoc?

If so, please either use a snippet or only show it once.